crates/proto/src/rr/rdata/svcb.rs

```diff
@@ -1032,7 +1032,7 @@ pub fn read(decoder: &mut BinDecoder<'_>, rdata_length: Restrict<u16>) -> ProtoR
     let svc_priority = decoder.read_u16()?.unverified(/*any u16 is valid*/);
     let target_name = Name::read(decoder)?;
 
-    let mut remainder_len = rdata_length.map(|len| len as usize - (decoder.index() - start_index)).unverified(/*valid len*/);
+    let mut remainder_len = rdata_length.map(|len| len as usize).checked_sub(decoder.index() - start_index).map_err(|len| format!("Bad length for RDATA of SVCB: {}", len))?.unverified(/*valid len*/);
```

Comment:
Do we not run rustfmt in CI? This line is clearly too long...

Reply:
rustfmt is run in CI as part of the test/compatibility. For some reason rustfmt has no problem with the length of this line (?)

Reply:
hm... the only thing we have in the rustfmt config file is the edition. don't know why it allows that line.

Reply:
it seems to come form the inline comment inside the call to unverified, if I remove that, rustfmt format as it should.